app/api/discount_codes.py

```diff
@@ -23,14 +23,27 @@ class Meta:
         self_view_kwargs = {'id': '<id>'}
         inflect = dasherize
 
-    @validates_schema
-    def validate_quantity(self, data):
-        if 'max_quantity' in data and 'min_quantity' in data:
-            if data['max_quantity'] < data['min_quantity']:
+    @validates_schema(pass_original=True)
+    def validate_quantity(self, data, orig_data):
+        o_data = orig_data['data']['attributes']
+        discount_code = DiscountCode.query.filter_by(id=orig_data['data']['id']).one()
+        if 'min_quantity' in o_data:
+            min_quantity = o_data.get('min_quantity')
+        else:
+            min_quantity = discount_code.min_quantity
+        if 'max_quantity' in o_data:
+            max_quantity = o_data.get('max_quantity')
+        else:
+            max_quantity = discount_code.max_quantity
```

Comment:
umm..yeah. Let it be where it is currently. And rename orig_data to something like `original` or `original_data`